functions/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../tsconfig.json",
   "compilerOptions": {
     "outDir": "../dist/functions",
-    "rootDir": "."
+    "rootDir": ".",
+    "lib": ["esnext.asynciterable", "dom"]
```

Comment:
The `dom` APIs won't be available in the function context, so you should remove that.

Reply:
Without `dom` included in the functions tsconfig when running watch the build will fail due to an issue with `tsc --watch` and `apollo-server` included types.

Reply:
Interesting. What's the error?

Reply:
Removed it to try to reproduce the issue. Not occuring. But what was happening before was the functions `tsc --watch` rebuild was failing due to not having `esnext.asynciterable`. It isn't happening now. Removed libs from `functions/tsconfig.json`